test/index.test.js

```diff
@@ -0,0 +1,60 @@
+var should = require('should');
+var sinon = require('sinon');
+
+var ga = require('../src/index');
+
+var GLOBALS = ['ga', 'GoogleAnalyticsObject', 'window', 'document'];
+
+describe('react-ga', function() {
+  function getGaCalls() {
+    var q = (window.ga && window.ga.q) || [];
+
+    return q.map(function(args) {
+      return [].slice.call(args);
+    });
+  }
+
+  beforeEach(function() {
+    sinon.stub(console, 'warn');
+    global.window = global;
+    global.document = {
+      createElement: function(name) {
+        return {};
+      },
+      getElementsByTagName: function(name) {
```

Comment:
We're faking the `window` and `document` objects here via node's [`global`](https://nodejs.org/api/globals.html#globals_global) object so that the GA initialization code runs.

An alternative would be to only run these tests in the browser, since that's the only environment where `react-ga` will actually run, but since the code has minimal dependencies on the browser environment, I thought it might be easier and faster to just test things in a node environment and stub out the rest.

**One thing this won't catch, however, is any advanced ES5 features we decide to use that older browsers don't support.** In the worst case, though, we could probably make minimal modifications to the test suite that allow it to run in both browser *and* node environments, though at that point we'd likely also have to swap out `should` for [`expect`](https://github.com/Automattic/expect.js/blob/master/README.md), since the former requires a fairly modern browser while the latter runs on IE6.